README.md

```diff
@@ -68,21 +71,23 @@ In the project npm root path create file "translate-config.json"
 ```
 
 {
-  "translationFile": "translations.json",
+  "translationsFile": "translations.json",
   "sheetId": "<sheetId>",
-  "sheetName": "<sheetName>"
+  "sheetName": "<sheetName>",
+  "locales": ["th", "en"],
+  "translationsPath": "src/lang"
 }
 
 ```
 
 
 | Key  | Description |
 | ------------- | ------------- |
-| `translationFile`  | `Path to file containing translations`  |
+| `translationsFile`  | `Path to file containing translations`  |
```

Comment:
This is the merged file I suppose or the intermediate file. Do we need this in config?

Reply:
Not really, we can omit this key. @romitamgai